Exiled.API/Features/Player.cs

```diff
@@ -1408,11 +1418,37 @@ public static Player Get(string args)
         /// <seealso cref="GiveReservedSlot()"/>
         public static bool AddReservedSlot(string userId) => ReservedSlot.Users.Add(userId);
 
+        /// <summary>
+        /// Adds a player's UserId to the list of reserved slots permanently. It will add a UserId to UserIDReservedSlots.txt file.
+        /// </summary>
+        /// <param name="userId">The UserId of the player to add.</param>
+        public static void PermanentlyAddReservedSlot(string userId) => ReservedSlots.Add(userId);
```

Comment:
Maybe instead of a different method, we can add a `bool permanent` to `AddReservedSlot`?

Reply:
Oh, that was the original intention. Thanks, Yamato